docs/views/includes/scripts.html

```diff
@@ -1,6 +1,10 @@
 <!-- Javascript -->
 <script src="/public/javascripts/jquery-1.11.3.js"></script>
-<script src="/node_modules/govuk-frontend/all.js"></script>
+
+{% for scriptUrl in additionalScripts %}
```

Comment:
where is `additionalScripts` coming from? Can't see another reference in the PR

Reply:
Oh dear, incomplete refactor.  It should be `pluginConfig.scripts` (which should also be renamed `extensionConfig` from https://github.com/alphagov/govuk-prototype-kit/pull/613/files#diff-78c12f5adc1848d13b1c6f07055d996eR122